src/index.js

```diff
@@ -60,6 +60,23 @@ class PaperWallet {
     return { claimData: claimData, qr: qr, version: ver }
   }
 
+  /**
+   * creates the wallet
+   * @param {any} vc
+   * @param {any} resultLink
+   */
+  async createWalletVc (vc, resultLink) {
+    console.log('Inside function', this)
```

Comment:
If we want to do some logging here, we should be using loglevel, but since we don't seem to have this as a dependency, I'd be good with just removing this.